### PR TITLE
Pipeline fixes

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

**Does this PR introduce a breaking change?**

```
- [ ] Yes
- [ ] No
```

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1 @@
+npm test
```
Comment (linusnorton) [inline .husky/pre-commit:1]: lint?

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -43,28 +43,21 @@ def uploadCoreCaseDataDefinitions(env, version) {
...
-    uploadCoreCaseDataDefinitions('demo', 'pr-615-3d45757c')
-  }
-
-  afterAlways('smoketest:preview') {
-    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'smoke-output/**'
-  }
-
   before('smoketest:aat') {
-    uploadCoreCaseDataDefinitions('aat', 'latest')
+    uploadCoreCaseDataDefinitions('aat', 'pr-615-3d45757c')
```
Comment (linusnorton) [inline Jenkinsfile_CNP:52]: is this still a thing?

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -43,28 +43,21 @@ def uploadCoreCaseDataDefinitions(env, version) {
...
   }
 
   afterAlways('smoketest:aat') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'smoke-output/**'
   }
-}
+
+  afterAlways('functionalTest:aat') {
+    sh 'exit 1'
```
Comment (linusnorton) [inline Jenkinsfile_CNP:60]: hashtag winning

**Review by linusnorton — APPROVED** [review]

File: Jenkinsfile_CNP (hunk 2 above)
Comment (velimd) [reply Jenkinsfile_CNP:52]: for some reason there isnt any latest. i check manually. ill look into it but it doesnt really matter as my other changes makes replaces this. 
https://github.com/hmcts/cmc-ccd-domain/pull/618/files#diff-a31d2a7e600e89b405cd675bd2f5bc6371e64b20cab9d32ab2a0279209a1882fR49

Comment (velimd) [reply Jenkinsfile_CNP:52]: just added it as im curious to see if this would work

**Review by linusnorton — APPROVED** [review]

**Review by damongreen123 — APPROVED** [review]

**Pull request merged**